pkg/daemon/daemon.go

```diff
@@ -186,7 +186,8 @@ const (
 
 	// Where nmstate writes the link files if it persisted ifnames.
 	// https://github.com/nmstate/nmstate/blob/03c7b03bd4c9b0067d3811dbbf72635201519356/rust/src/cli/persist_nic.rs#L32-L36
-	systemdNetworkDir = "etc/systemd/network"
+	systemdNetworkDir   = "etc/systemd/network"
+	nmstatePersistStamp = ".nmstate-persist.stamp"
```

Comment:
This is another thing I don't think the MCO should know about; it interlocks the implementation of these two things.

I would really like to go back to a plan where the MCD just invokes a binary and doesn't have much network specific code at all.

In other words, the sole "API" or contract between the two is invoking a binary.